content/v2.0/query-data/flux/sql.md

```diff
@@ -154,6 +155,77 @@ Use the variable to manipulate queries in your dashboards.
 
 ---
 
+## Use secrets to populate connection credentials
+If your SQL database requires authentication, use [InfluxDB secrets](/v2.0/security/secrets/)
+to store and populate sensitive connection credentials for your database.
+By default, InfluxDB encrypts and stores secrets in its internal key-value store, BoltDB,
+but also integrates directly with [Vault](https://www.vaultproject.io/).
```

Comment:
```
By default, InfluxDB stores secrets in its internal key-value store, BoltDB.
For added security, use [Vault](https://www.vaultproject.io/).
```

Reply:
(BoltDB does not encrypt data, so let's encourage people to use Vault for projects that involve authentication.)